### PR TITLE
Update integrations

### PR DESCRIPTION
**Pull request by dafimor**
@refaelos can you pls review?

### CODE REVIEW

**Comment by refaelos** [comment]
@dafimor,  if @borisspektor reviewed you can push.

**Comment by dafimor** [comment]
@borisspektor pls review

File: src/documents/cocos2dx/cpp/Grow_Sync.html.md [hunk 1]
```diff
@@ -41,6 +42,49 @@ With State & Economy Sync you can:
 
 <div class="warning-box">Make sure to start gameplay or change local state only **after** [`EVENT_STATE_SYNC_FINISHED`](/cocos2dx/cpp/Grow_Sync#OnStateSyncFinished) event is triggered. </div>
 
+## Conflict Resolution
+
+Cross device syncing allows the player to have the same game state across all of his/her devices, this means that if the player plays on one device and reaches Level 10, this new state is saved in the server. The next time this player plays from a different device the saved state will be updated to his/her device and the game will continue from Level 10.
```
Comment (borisspektor) [inline src/documents/cocos2dx/cpp/Grow_Sync.html.md]: @dafimor maybe:
Cross device syncing allows **players** to have the same game state across all of **their** devices. This means that if **a** player.....

**Comment by borisspektor** [comment]
@dafimor looks good to me

**Pull request merged**